src/khiva/matrix.cpp

```diff
@@ -6,238 +6,20 @@
 
 #include <khiva/matrix.h>
 #include <khiva/normalization.h>
+
```

Comment:
I am not sure why is here this blank line? Do you want to differenciate between the header of khiva and others? If that is the case, don't remove it.